### ISSUE TITLE
Mac preferences direction

### ISSUE THREAD

**Issue by DavidJJ** [StudioPetrikas/FreeCADLink_VertUI#8]
Can you add the mac preferences directory to the instructions for us Mac Users.
This is the path:
~/Library/Preferences/FreeCAD

**Comment by luzpaz** [comment]
@DavidJJ do you know how to submit a PR ?

**Comment by DavidJJ** [comment]
Yes. But for a 1 line documentation change I didn't think it was worth it.  

VertUI on Mac (M1 16" top of the line proMax) drops performance to the point that it takes multiple seconds for UI interactions to show up.

**Comment by StudioPetrikas** [comment]
@DavidJJ Interesting. There's not really much that can interfere that way, since it's only a stylesheet. Could this be ARM-related?

**Comment by DavidJJ** [comment]
> @DavidJJ Interesting. There's not really much that can interfere that way, since it's only a stylesheet. Could this be ARM-related?

Yeah, I got nothing.  I looked through the "code" but as you said it's all just style sheets. But as soon as it is applied performance drops to completely unusable.  Like "click takes 10 seconds" unusable while stock setup is instantaneously fast.
